RELEASE-NOTES.md

```diff
@@ -1,5 +1,62 @@
 # Prefect Release Notes
 
+## Release 2.13.3
+
+## Allow configuration of a work pool's base job template via the CLI
+Previously, the creation and modification of work pools, including editing the base job template, were done through the Prefect UI. Now you can alter the base job template through CLI commands:
+
+Retrieve the default base job template for a given work pool:
+```bash
+prefect work-pool get-default-base-job-template --type kubernetes
+```
+
+You can customize the base job template by passing a JSON file to the `--base-job-template` flag:
+```bash
+prefect work-pool create my-k8s-pool --type kubernetes --base-job-template ./path/template.yaml
+```
+
+Useful for version control, you can now make updates to a work pool's base job template via the CLI:
+```bash
+prefect work-pool update my-work-pool --base-job-template base-job-template.json --description "My work pool" --concurrency-limit 10
+```
+
+See the documentation on [work pools](https://docs.prefect.io/latest/concepts/work-pools/) for more information, or see the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/10793
+- https://github.com/PrefectHQ/prefect/pull/10797
+- https://github.com/PrefectHQ/prefect/pull/10796
+- https://github.com/PrefectHQ/prefect/pull/10798
+- https://github.com/PrefectHQ/prefect/pull/10844
+
+## Allow users to customize their default flow runs view in the Prefect UI
+You can now set your own default filter view on your Flow Runs page! You must first save and name a view before you can set it as your default.
+Finally, be aware that this setting is only stored locally so it will not be shared across machines/browsers. 
```

Comment:
```suggestion
You can now set your own default filter view on your Flow Runs page! You must first save and name a view before you can set it as your default. This setting is only stored locally so it will not be shared across machines/browsers. 
```